src/PKSim.Presentation/Presenters/ContextMenus/MultipleEventNodeContextMenu.cs

```diff
@@ -0,0 +1,35 @@
+﻿using System.Collections.Generic;
+using OSPSuite.Presentation.Nodes;
+using OSPSuite.Presentation.Presenters;
+using OSPSuite.Presentation.Presenters.ContextMenus;
+using PKSim.Core;
+using PKSim.Core.Model;
+
+namespace PKSim.Presentation.Presenters.ContextMenus
+{
+   public class MultipleEventNodeContextMenu : MultipleBuildingBlockNodeContextMenu<PKSimEvent>
```

Comment:
I had to create one context menu for each type as a special call was required for Compound. That way, we can change the multiselect menu based on building block types if we feel like iit